MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: post
+title: Data annotations in Maui DataForm control | Syncfusion
+description: Learn here all about Data annotations support in Syncfusion Maui DataForm (SfDataForm) control and more.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+# Data annotations in Maui DataForm (SfDataForm)
+
+The data form supports the following attribute, and these attributes can be accessible using `System.ComponentModel.DataAnnotation` assembly.
+
+## Display attribute
+
+<table>
+<tr>
```

Comment:
Add example for all attribute properties

Reply:
examples are included